clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheResourceDefinition.java

```diff
@@ -109,6 +108,7 @@ public AttributeDefinition getDefinition() {
         @Deprecated BATCHING("batching", ModelType.BOOLEAN, builder -> builder.setDefaultValue(new ModelNode(false)), InfinispanModel.VERSION_3_0_0),
         @Deprecated INDEXING("indexing", ModelType.STRING, builder -> builder.setDefaultValue(new ModelNode(Index.NONE.name())).setValidator(new EnumValidator<>(Index.class)), InfinispanModel.VERSION_4_0_0),
         @Deprecated INDEXING_PROPERTIES("indexing-properties", InfinispanModel.VERSION_4_0_0),
+        @Deprecated JNDI_NAME("jndi-name", ModelType.STRING, UnaryOperator.identity(), InfinispanModel.VERSION_5_0_0),
```

Comment:
The deprecated attribute at CacheContainerResourceDefinition.java uses InfinispanModel.VERSION_6_0_0 while this one uses InfinispanModel.VERSION_5_0_0. is that correct?

Reply:
Ah - that's a typo.  Will fix.